experimental/NeuralNets/Lib.idr

```diff
@@ -0,0 +1,151 @@
+module Lib
+import Prelude.Doubles as db
+
+export
+stopwords : List String
+stopwords = ["the", "a", "and", "is", "be", "will", "is"] 
+
+export
+nums : String
+nums = "0123456789!@#$%^&*()_+=-:;"
+
+export
+removeStopWords : String -> List String
+removeStopWords line = do
+              let tmp = words line
+              foldr (\x, acc => if elem x stopwords then acc else x::acc ) [] tmp
+
+export
+makeWordPairs : List String -> List (String, String)
+makeWordPairs [] = []
+makeWordPairs (x::y::z::h::xs) = (x,y)::(x,z)::(y,z)::(y,h)::(z,x)::(z,y)::(z,h)::(h,x)::(h,y)::(h,z)::makeWordPairs xs
+makeWordPairs (x::y::z::xs) = (x,y)::(x,z)::(y,z)::(z,x)::(z,y)::makeWordPairs xs
+makeWordPairs (x::y::xs) = (x,y)::makeWordPairs xs
```

Comment:
Comments are welcome as soon as functions are not trivial, this one is kinda on the edge.  It looks like it's making pairs of between neighboring words, not completely sure.

Reply:
Will add comments here.